<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="central_widget_frame">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 220, 215)

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="background_frame">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="content_frame">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QFrame" name="top_frame">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 0, 0);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="title_label">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>14</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: white
</string>
                 </property>
                 <property name="text">
                  <string>PERSONAL MANTENIMIENTO</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QFrame" name="buttons_holder_frame">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <widget class="QToolButton" name="minimize_button">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>0</y>
                    <width>21</width>
                    <height>22</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../icons/minimize-window.png</normaloff>../../icons/minimize-window.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                 <widget class="QToolButton" name="restore_button">
                  <property name="geometry">
                   <rect>
                    <x>50</x>
                    <y>0</y>
                    <width>21</width>
                    <height>22</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../icons/restore-window.png</normaloff>../../icons/restore-window.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                 <widget class="QToolButton" name="maximize_button">
                  <property name="geometry">
                   <rect>
                    <x>50</x>
                    <y>0</y>
                    <width>21</width>
                    <height>22</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../icons/maximize-window.png</normaloff>../../icons/maximize-window.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                 <widget class="QToolButton" name="close_button">
                  <property name="geometry">
                   <rect>
                    <x>90</x>
                    <y>0</y>
                    <width>21</width>
                    <height>22</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../icons/close-window.png</normaloff>../../icons/close-window.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="action_frame">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>39</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QPushButton" name="new_recipe_button">
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Rockwell</family>
                   <pointsize>10</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	background-color: rgb(170, 0, 0);
	color: white
}
QPushButton::hover {background-color: rgb(0, 170, 255)};</string>
                 </property>
                 <property name="text">
                  <string>Añadir Personal</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../../icons/plus.png</normaloff>../../icons/plus.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="minimumSize">
                    <size>
                     <width>30</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>30</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: white;
border-radius: 0px;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap>../../icons/search.png</pixmap>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="serch_line_edit">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>500</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: white;
border-radius: 0px;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="placeholderText">
                    <string>Buscar Personal</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources_from_qt.qrc"/>
 </resources>
 <connections/>
</ui>
